### ISSUE TITLE
Conditional Variable Othertext

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-transform#162]
Cicero `draft` will be changed to put all conditional variables in markdown XML

This package should handle this change and put variables in the Slate object, maybe in the data of the inline?

For `commonMark` to `ciceroMark` and `ciceroMark` to Slate:
Adjust to handling these inlines being changed

Related issue [comment by Jerome][jeromeComment]:

```
Late Delivery and Penalty.
----

In case of delayed delivery<if id="forceMajeure" whenTrue=" except for Force Majeure cases," whenFalse=" even when there is a Force Majeure," value="true"/>
<variable id="seller" value="%22Dan%22"/> (the Seller) shall pay to <variable id="buyer" value="%22Steve%22"/> (the Buyer) for every <variable id="penaltyDuration" value="2%20days"/>
of delay penalty amounting to <variable id="penaltyPercentage" value="10.5"/>% of the total value of the Equipment
whose delivery has been delayed. Any fractional part of a <variable id="fractionalPart" value="days"/> is to be
considered a full <variable id="fractionalPart" value="days"/>. The total amount of penalty shall not however,
exceed <variable id="capPercentage" value="55.0"/>% of the total value of the Equipment involved in late delivery.
If the delay is more than <variable id="termination" value="15%20days"/>, the Buyer is entitled to terminate this Contract.
```

```json
          {
            "object": "inline",
            "type": "conditional",
            "data": {
              "id": "forceMajeure",
              "whenTrue": " except for Force Majeure cases,",
              "whenFalse": " even when there is a Force Majeure,",
              "value": false
            },
            "nodes": [
              {
                "object": "text",
                "text": " even when there is a Force Majeure,",
                "marks": []
              }
            ]
          }, 
```

[jeromeComment]: https://github.com/accordproject/cicero-ui/issues/233#issuecomment-582957381

**Comment by jeromesimeon** [comment]
Note: the sample was done by hand, the text content for `whenTrue` and `whenFalse` should be URL-encoded. (@dselman )

**Comment by jeromesimeon** [comment]
This was fixed in #180
